docs/staking-modules/csm/contracts/CSEjector.md

```diff
@@ -0,0 +1,153 @@
+# CSEjector
+
+- [Source code](https://github.com/lidofinance/community-staking-module/blob/v2.0/src/CSEjector.sol)
+- [Deployed contract](https://etherscan.io/address/0xc72b58aa02E0e98cF8A4a0E9Dce75e763800802C)
+
+`CSEjector.sol` is a supplementary contract responsible for interactions with EIP-7002-powered Lido Withdrawal credentials via `VEB`. Node Operators can voluntarily eject their validators. `CSStrikes.sol` uses `CSEjector.sol` to trigger exits for validators that have surpassed the strike threshold.
```

Comment:
```suggestion
`CSEjector.sol` is a supplementary contract responsible for interactions with EIP-7002-powered Lido Withdrawal credentials via `TriggerableWithdrawalsGateway` (`TWG`). Node Operators can voluntarily eject their validators. `CSStrikes.sol` uses `CSEjector.sol` to trigger exits for validators that have surpassed the strike threshold.
```

Reply:
AFAIK `CSEjector` is provided with the `ADD_FULL_WITHDRAWAL_REQUEST_ROLE` to interact directly with TWG though